determined-ai/determined
# feat: Enable username and display name search on cluster "user" list [WEB-1009]

## Description

Enables the ability to search by `username` and `display name` in the cluster user list.

https://user-images.githubusercontent.com/103522725/224180887-03718d94-8fbf-4d5f-85d3-a4c76e4c54c8.mov

Previously it was thought that the ability to filter by username or display name in the `WorkspaceMembers` page existed but that was not the case. The search worked only on the display name if it existed, otherwise it worked on just the username.  The change needed for the `WorkspaceMembers` page has been made in the PR here: https://github.com/determined-ai/determined-ee/pull/733

https://user-images.githubusercontent.com/103522725/224180866-338e5456-94d4-4ae8-bfc0-02d1cdc78ee7.mov








## Test Plan

1. Navigate to the user administration page in an `RBAC` enabled cluster.
2. Using the search icon confirm that the search works on both `username` and `display name` for users. A user with match in either field show show up in the search results.
3. Ensure table sorting still works with a `name` filter applied.





## Commentary (optional)




## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
WEB-1009
